src/DqtApi/Validators/GetTeacherValidator.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using DqtApi.Models;
+using FluentValidation;
+
+namespace DqtApi.Validators
+{
+    public class GetTeacherValidator : AbstractValidator<GetTeacherRequest>
```

Comment:
Is this used?

Reply:
It's going to be - leaving it in for now